test/quadquizaminos/qna_test.exs

```diff
@@ -4,9 +4,9 @@ defmodule Quadquizaminos.QnaTest do
   alias Quadquizaminos.QnA
 
   test "answers are determined by hyphen or asterisk in newline" do
-    %{answers: answers} = QnA.question("open_chain_sample")
+    answers = QnA.question(["qna"], "open_chain_sample")
```

Comment:
The arguments passed here is different with what question/2 takes . Its supposed to take in the category and position
```
  def question(category, position) do
    build(category, position)
  end

```